force-app/main/default/classes/HOT_KrrCalloutQueuable.cls

```diff
@@ -76,6 +67,7 @@ public without sharing class HOT_KrrCalloutQueuable implements Queueable, Databa
                 person.Id = personId;
                 person.INT_LastUpdatedFromKRR__c = System.now();
                 person.INT_KrrIntegrationStatus__c = 'Failed';
+                person.HOT_IntegrationErrorCounter__c++;
```

Comment:
Vet ikke om du kan si ++ her, ettersom person.HOT_IntegrationErrorCounter__c ikke er deklartert. Vi har jo ikke henta ut noen verdi for denne... Kanskje vi rett og slett ikke kan telle opp slik.